cypress.config.ts

```diff
@@ -44,6 +44,14 @@ export default defineConfig({
         },
       );
 
+      config.env.CHAPTER_1_OWNER = jwt.sign(
```

Comment:
```suggestion
      config.env.JWT_CHAPTER_1_ADMIN = jwt.sign(
```
Also, we don't need two tokens for the same user, so could you delete `JWT_ADMIN_USER`?